src/blockfield.js

```diff
@@ -0,0 +1,54 @@
+let redact;
+
+const BLOCKFIELD_RE = /^{([^}]+)}/;
+const BLOCKFIELD = 'blockfield';
+
+module.exports = function blockfield() {
+  if (this.Parser) {
```

Comment:
When would this not be present?

Reply:
In practice? Probably never. But in theory, it's entirely valid to set up a unified pipeline without a parser; for example, when you already have a parsed syntax tree and just want to render it. It's good practice to have this plugin be a no-op in that scenario.

Note that this check is also in the [original version of the plugin](https://github.com/code-dot-org/code-dot-org/blob/939f1ac/bin/i18n/plugins/blockfield.js)